dotnet/src/IntegrationTests/Connectors/Memory/Postgres/PostgresVectorStoreRecordCollectionTests.cs

```diff
@@ -0,0 +1,525 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.VectorData;
+using Xunit;
+
+namespace SemanticKernel.IntegrationTests.Connectors.Memory.Postgres;
+
+[Collection("PostgresVectorStoreCollection")]
+public sealed class PostgresVectorStoreRecordCollectionTests(PostgresVectorStoreFixture fixture)
```

Comment:
It would be nice to have a test that reads some data that wasn't upserted by the Collection.  E.g. if you create a table and upsert some records using SQL, and then read them using the Collection.  The reason being that we want to ensure that the data in the DB is actually what we want it to be.  E.g. if we do two complimentary things wrong, on upsert and read, so it looks fine when using the Collection, but the data isn't stored in the way we need it to in the DB, a test where we didn't write the data via the Collection should catch this.